core/common/assets/js/views/modal/header.js

```diff
@@ -38,7 +38,8 @@ export default class extends Marionette.LayoutView {
 
 		if (
 			window.elementor && elementor.config?.admin_conversion_center_contact_url &&
-			'contact-buttons' === elementor.config?.document?.type
+			'contact-buttons' === elementor.config?.document?.type &&
+			this.$el.find( '#elementor-template-library-header-menu' ).length
```

Comment:
Can we remove this change?

Reply:
@louiswol94 this is needed as otherwise we close the display condition modal like we close the library modal, basically we return to the admin dashboard. This is not the cleanest approach, but closing the modal doesn't fire a command, which is why we have this code here. And the only difference between the two modals, without targetting a string which is prone to failure, is that the library modal had tabs, while the display conditions doesn't have them. If you have suggestions let me know.

Reply:
@nicoladj77 I see. So closing the Display Conditions modal will also redirect to Admin?

Can you please extract these conditions to a method for readability? `isFloatingButtonLibraryClose`? Something like that, so we know what the situation is you are checking for.